tools/pythonpkg/tests/fast/spark/test_spark_functions_date.py

```diff
@@ -224,3 +224,10 @@ def test_add_months(self, spark):
         assert result[0].with_literal == date(2024, 6, 12)
         assert result[0].with_str == date(2024, 7, 12)
         assert result[0].with_col == date(2024, 7, 12)
+
+    def test_try_to_timestamp(self, spark):
+        df = spark.createDataFrame([('1997-02-28 10:30:00',), ('2024-01-01',), ('invalid')], ['t'])
+        res = df.select(F.try_to_timestamp(df.t).alias('dt')).collect()
```

Comment:
This only tests the path where `format` is None, not the path where an explicit format is given